client/src/components/Button/Button.css

```diff
@@ -1,3 +1,81 @@
-button {
-	background-color: red;
+.button-light-pink {
+    border: 1px;
+    background-color: #FAE5DC;
+    color: #441F0E;
+    font-size: 15px;
+    font-weight: bold;
+    line-height: 22px;
+    font-style: normal;
+    font-family: Sahitya;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    height: 31px;
+    width: 88px;
+   
 }
+
+.button-dark-pink {
+    font-family: Raleway;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 15px;
+    line-height: 18px;
+    background-color: rgba(68, 31, 14, 0.2);
+    border: 1px;
+    height: 28px;
+    width: 75px;
+}
+
+.button-small-grey {
+    font-family: Sahitya;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 18px;
+    line-height: 28px;
+    color: #000000;
+    background: rgba(68, 31, 14, 0.2);
+    height: 28px;
+    width: 75px;
+    border: 1px;
+}
+
+.button-large-red {
+    font-family: Raleway;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 14px;
+    line-height: 16px;
+    background-color: #441F0E;
+    border: 0.3px solid #441F0E;
+    box-sizing: border-box;
+    color: white;
+    width: 138px;
+    height: 30px;
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
```

Comment:
This should be broken up. We want to be able to have a button of any color and size combination. Doing each combination manually is going to be fragile and a lot of work. Can you figure out a better way to do this?